src/elfpy/wallet.py

```diff
@@ -16,21 +16,23 @@ class Wallet:
 
     # fungible
     address: int
-    base_in_wallet: float
-    lp_in_wallet: float = 0  # they're fungible!
+    base: float
+    lp: float = 0  # pylint: disable=invalid-name
```

Comment:
what if you just made this a valid name? like liquidity?

Reply:
Liquidity doesn't really have the same meaning to be honest. The linting directive doesn't really bother me.